TestCases/optimization_euler/equivalentarea_naca64206/NACA64206_direct.cfg

```diff
@@ -0,0 +1,160 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%                                                                              %
+% SU2 configuration file                                                       %
+% Case description: NACA64-206 in inviscid, supersonic flow                  %
+% Author: Yuki Utsumi	      	      	              	       	       %
+% Institution: Individual				               %
+% Date: 2021.07.21                                                            %
+% File Version 7.1.1 "Blackbird"                                                %
+%                                                                              %
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+% ------------- DIRECT, ADJOINT, AND LINEARIZED PROBLEM DEFINITION ------------%
+%
+% Physical governing equations (EULER, NAVIER_STOKES)
+SOLVER= EULER
+%
+% Mathematical problem (DIRECT, CONTINUOUS_ADJOINT, DISCRETE_ADJOINT)
+MATH_PROBLEM= DIRECT
```

Comment:
```suggestion
```
If this is the only difference between the two config files you can remove it.
If MATH_PROBLEM is omited, SU2 deduces the type of problem from the executable, SU2_CFD -> DIRECT, SU2_CFD_AD -> DISCRETE_ADJOINT

Reply:
I have consolidated two configuration files into one.